plugins/Goals/templates/getGoalReportView.twig

```diff
@@ -37,6 +37,10 @@
             </li>
         {% endif %}
     </ul>
+
+    <a href="javascript:;" class="segmentedlog" onclick="DataTable_RowActions_Registry.getActionByName('SegmentVisitorLog').createInstance({}).showVisitorLog('Goals.getMetrics', 'visitConvertedGoalId=={{ idGoal }}', {})">
```

Comment:
Could we maybe instead provide a static function `SegmentedVisitorLog.show(method, segment)` or similar? This way we could reuse it somewhere else more easily and would hide the dependency to RowActions etc.

Even better would be maybe if `DataTable_RowActions_SegmentVisitorLog.prototype.doOpenPopover` used this method as well so we do not use RowActions here at all